src/aat/resources/features/F-1008 - Submit Event Creation Handle Case Links/F-1008_Test_Data_Base.td.json

```diff
@@ -0,0 +1,50 @@
+{
+	"_guid_": "F-1008_Test_Data_Base",
+
+	"productName": "CCD Data Store",
+	"operationName": "Submit event creation as Case worker",
+
+	"method": "POST",
+	"uri": "/caseworkers/{uid}/jurisdictions/{jid}/case-types/{ctid}/cases/{cid}/events",
```

Comment:
If a **common TD file** to cover the basics of the API signature does not exist can we create on and use it as this endpoint is being used multiple times in these tests:

* F-1008_Test_Data_Base

and see also the following in hmcts/ccd-data-store-api#1719:

* F-1006_Test_Data_Base
* F-1008_Test_Data_Base
* F-1008_Test_Data_Base_Negative

Reply:
Done [c129a9a](https://github.com/hmcts/ccd-data-store-api/pull/1872/commits/c129a9a6a6fdff4226e65e0b027691d3cd49352f)